docs/trajectory_gradspec.rst

```diff
@@ -0,0 +1,29 @@
+=========================================
+ The trajectory binary file specification
+=========================================
+
+The k-space trajectories are transformed to a binary file which is processed by scanner.
+This file mainly specifies an arbitrary gradient profile which is played on scanner at gradient raster time rate (10e-6 seconds).
+
+The binary file format is specified as follows:
+
+
+|Name          |Type |Size   |Unit   |Description                                                              |
+|--------------|-----|-------|-------|-------------------------------------------------------------------------|
+|Version       |FLOAT|1      |n.a.   |file version this new version would be “4.0”                             |
```

Comment:
Add a note on the different version numbers, and specify which version mri-nufft supports.